addons/web/static/tests/views/list_view_tests.js

```diff
@@ -18299,4 +18299,39 @@ QUnit.module("Views", (hooks) => {
         await click(document, "body .modal footer button.btn-primary");
         assert.verifySteps(["a"]);
     });
+
+    QUnit.test("x2many onchange, check result", async function (assert) {
+        assert.expect(6);
+
+        serverData.models.foo.onchanges = {
+            m2m: function () {},
+        };
+
+        await makeView({
+            type: "list",
+            resModel: "foo",
+            serverData,
+            arch: `<tree editable="bottom" expand="1">
+                    <field name="m2m" widget="many2many_tags"/>
+                    <field name="m2o"/>
+                </tree>`,
+            async mockRPC(route, args) {
+                if (args.method === "onchange") {
+                    assert.step("onchange");
+                    await nextTick();
+                    return {"value": {"m2o": [3, "Value 3"]}};
+                }
+            },
+        });
+
+        assert.strictEqual(target.querySelector(".o_data_row .o_many2many_tags_cell").textContent, "Value 1Value 2");
+        assert.strictEqual(target.querySelector(".o_data_row .o_list_many2one").textContent, "Value 1");
+        await click(target.querySelector(".o_data_row .o_many2many_tags_cell"));
+        await selectDropdownItem(target, "m2m", "Value 3");
+
+        assert.verifySteps(["onchange"]);
+        await click(target.querySelector(".breadcrumb"));
```

Comment:
?

Reply:
If this is to switch the row back in readonly, I suggest to click on Save which is more obvious